kpops/components/streams_bootstrap_v3/streams/streams_app.py

```diff
@@ -0,0 +1,155 @@
+import logging
+from functools import cached_property
+
+from pydantic import Field, computed_field
+from typing_extensions import override
+
+from kpops.component_handlers.kubernetes.pvc_handler import PVCHandler
+from kpops.components.base_components.helm_app import HelmApp
+from kpops.components.base_components.kafka_app import KafkaAppCleaner
+from kpops.components.common.app_type import AppType
+from kpops.components.common.topic import KafkaTopic
+from kpops.components.streams_bootstrap_v3.base import (
+    StreamsBootstrapV3,
+)
+from kpops.components.streams_bootstrap_v3.streams.model import (
+    StreamsAppValues,
+)
+from kpops.utils.docstring import describe_attr
+
+log = logging.getLogger("StreamsApp")
+
+STREAMS_BOOTSTRAP_V3 = "3.0.0"
```

Comment:
can't we import it from streams_bootstrap_v3/base?

Reply:
Improted